### ISSUE TITLE
Add a CI job to check openapi and swagger 

### ISSUE THREAD

**Issue by Shelley-BaoYue** [kubeedge/kubeedge#5679]
**What would you like to be added/modified**: 

Add a CI job to check openapi and swagger 

**Why is this needed**:

As PR [#](https://github.com/kubeedge/kubeedge/pull/5668) has achieved auto-generate API docs. In this PR, we have add generate-openapi.sh and generate-swagger.sh to generate openapi and swagger. We need add a CI job to check this item.

**Comment by EraKin575** [comment]
@Shelley-BaoYue can I work on this issue? Please assign me this issue

**Comment by Shelley-BaoYue** [comment]
@EraKin575  Of course, please go ahead. Feel free to communicate here if you have any question. 
/assign @EraKin575

**Comment by EraKin575** [comment]
Thanks. @Shelley-BaoYue !I have to add a CI-Job to test if the openapi definitions and specifications are generated or not, right?

**Comment by Shelley-BaoYue** [comment]
In PR https://github.com/kubeedge/kubeedge/pull/5668, we can use the scripts to generate openapi and swagger file. But if we modified the API, these files won't be auto modified. We need to add a CI job to check whether these files need to be modified. If yes, the system prompts the developer to submit the corresponding updates. (I think you can ref to https://github.com/kubeedge/kubeedge/blob/master/hack/verify-crds.sh)

**Comment by EraKin575** [comment]
Thanks!

**Comment by EraKin575** [comment]
I am really sorry for such kind of inconvinience for opening multiple PRs
